packages/media-query/src/create-media-query.ts

```diff
@@ -1,12 +1,17 @@
-import { objectKeys, Dict, isNumber } from "@chakra-ui/utils"
+import { Dict, isNumber, breakpoints as bps } from "@chakra-ui/utils"
 import calculateMeasurement from "calculate-measurement"
 
 /**
  * Create a media query string from the breakpoints,
  * using a combination of `min-width` and `max-width`.
  */
 function createMediaQueries(breakpoints: Dict) {
-  const sorted = getSortedKeys(breakpoints)
+  // get the non-number breakpoint keys from the provided breakpoints
+  const keys = Object.keys(breakpoints)
```

Comment:
I see that there was an import of objectKeys function, wonder what is different from Object.keys (maybe some polyfills ?) But it might be worth using it here ?

Reply:
It was solely for wrapping `Object.keys` with more accurate types which we're not using here, so I think we're good!